### ISSUE TITLE
[14.0] widget=ace line numbers are overlapping its content (in read mode)

### ISSUE THREAD

**Issue by ivantodorovich** [odoo/odoo#61729]
**Impacted versions:**
14.0

**Steps to reproduce:**
1. Go to Settings > Scheduled Actions
2. Open Base: Auto-vacuum internal data

**Current behavior:**
Lines numbers are overlapping the widget content:
![image](https://user-images.githubusercontent.com/1914185/99092777-5b3b0b00-25b0-11eb-824d-bdd2bb4454b1.png)

Editing the record renders them properly:
![image](https://user-images.githubusercontent.com/1914185/99092834-73ab2580-25b0-11eb-8df7-c7c8e0ce5e12.png)

**Expected behavior:**
Self-explainatory

**Comment by Yenthe666** [comment]
Yep, I can confirm it. I've seen it a few times before too. It is also weird because in some cases when you edit it shows up and sometimes it doesn't.

**Comment by netosjb** [comment]
I report that I also came across this bug today.
